smtc-modem-cores/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "smtc-modem-cores"
+version = "0.1.3"
+authors = ["Louis Thiery <thiery.louis@gmail.com>"]
+edition = "2018"
```

Comment:
Use 2021 edition? (Here and other `Cargo.toml` files as well.)

Reply:
copy-pasted an old project to get my started! it shows :smile: